### PR TITLE
[434688, 505760] Fixed 'Show In/ Generated Code'

### PR DESCRIPTION
**Pull request by kthoms**
Provide adapter IShowInTargetList for the editor through adapter factory
Implemented IShowInTarget for DerivedSourceView
Removed bogus menu contribution (parameters are not supported)

Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

File: org.eclipse.xtend.ide/plugin.xml [hunk 1]
```diff
@@ -1571,4 +1559,14 @@
      		 </adapt>
       </definition>
    </extension>
+   <extension
+         point="org.eclipse.core.runtime.adapters">
+      <factory
+            adaptableType="org.eclipse.xtext.xbase.ui.editor.XbaseEditor"
```
Comment (cdietrich) [inline org.eclipse.xtend.ide/plugin.xml]: does this affect other editor like domain model?

Comment (kthoms) [reply org.eclipse.xtend.ide/plugin.xml]: Good point. the action will also appear for dmodel, but with empty contents. On the other side there is a "Open Generated File" action which opens the generated Java code in an editor. I would expect that it should also use the DerivedSourceView. Maybe we should move it to Xbase?

Comment (cdietrich) [reply org.eclipse.xtend.ide/plugin.xml]: If I remember correctly there is a bugzilla for that somewhere

Comment (cdietrich) [reply org.eclipse.xtend.ide/plugin.xml]: https://bugs.eclipse.org/bugs/show_bug.cgi?id=381839

Comment (cdietrich) [reply org.eclipse.xtend.ide/plugin.xml]: and drived resource view and ope generated file are not the same

Comment (kthoms) [reply org.eclipse.xtend.ide/plugin.xml]: I guess generalizing this to Xbase is a bigger issue. As a workaround we could create an XtendEditor deriving from XbaseEditor to have a more concrete target for now. What do you think?

Comment (cdietrich) [reply org.eclipse.xtend.ide/plugin.xml]: :+1:

**Comment by kthoms** [comment]
Updated. The action is now only available for Xtend editors.

**Comment by cdietrich** [comment]
Lgtm

**Pull request merged**